lib/ui/recovery_key_dialog.dart

```diff
@@ -136,6 +158,7 @@ class _RecoveryKeyDialogState extends State<RecoveryKeyDialog> {
     );
   }
 
+  // author civil bicycle drop gown ship file shallow chicken hub isolate among report piece cram crumble leopard guard spike segment update fall defy achieve
```

Comment:
This key is for test account on local machine, so no harm done 😥 . Will remove it.